workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -129,5 +130,214 @@ private static bool TryGetApkPath(string rootPath, out string apkPath)
             apkPath = string.Empty;
             return false;
         }
+
+        private static void LaunchiOS(bool shouldConnectLocally)
+        {
+            try
+            {
+                EditorUtility.DisplayProgressBar("Preparing your Mobile Client", "Building your XCode project", 0f);
+
+                var toolsConfig = GdkToolsConfiguration.GetOrCreateInstance();
+                if (string.IsNullOrEmpty(toolsConfig.DevelopmentTeamId))
+                {
+                    Debug.LogError("Development Team Id was not specified. Unable to build the XCode project.");
+                    return;
+                }
+
+                if (RedirectedProcess.Command("xcodebuild")
+                    .WithArgs("build-for-testing",
+                        "-project", Path.Combine(XCodeProjectPath, XCodeProjectFile),
+                        "-derivedDataPath", DerivedDataPath,
+                        "-scheme", "Unity-iPhone",
+                        $"DEVELOPMENT_TEAM={toolsConfig.DevelopmentTeamId}",
+                        "-allowProvisioningUpdates")
+                    .Run() != 0)
+                {
+                    Debug.LogError(
+                        $"Failed to build your XCode project. Make sure you have xcodebuild installed and check the logs.");
```

Comment:
Could another reason for this failing be that you haven't built the iOS target yet?

Reply:
Also where can I find the logs?

Reply:
they will be printed inside the Unity console

Reply:
and yes, it could be that it simply doesn't exist. will add a check for that